src/gsHSplines/gsHDomainBoundaryIterator.h

```diff
@@ -208,22 +208,22 @@ class gsHDomainBoundaryIterator: public gsDomainIterator<T>
             {
```

Comment:
I haven't found any place, where updateLeaf() is called. How do we know it is correct?